<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminstorAdd</class>
 <widget class="QWidget" name="AdminstorAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>342</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>519</width>
    <height>342</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_username_status">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sure">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>确认 </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>99</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>410</width>
     <height>133</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>131</width>
        <height>131</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="image_resource.qrc">:/new/prefix1/image_resource/Admin.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>271</width>
        <height>111</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 italic 30pt &quot;URW Chancery L&quot;;
color: rgb(30, 26, 188);</string>
      </property>
      <property name="text">
       <string>注册新管理员</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>340</width>
     <height>99</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 italic 13pt &quot;URW Chancery L&quot;;</string>
      </property>
      <property name="text">
       <string>新用户名</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_username">
      <property name="minimumSize">
       <size>
        <width>251</width>
        <height>27</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>71</width>
        <height>17</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 italic 13pt &quot;URW Chancery L&quot;;</string>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="minimumSize">
       <size>
        <width>251</width>
        <height>27</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>67</width>
        <height>17</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 italic 13pt &quot;URW Chancery L&quot;;</string>
      </property>
      <property name="text">
       <string>权限 </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox_role">
      <property name="minimumSize">
       <size>
        <width>261</width>
        <height>27</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="image_resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>AdminstorAdd</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
